src/MaterialEditor.Base/UI/UI.cs

```diff
@@ -546,7 +546,12 @@ void PopulateListMaterial(Material mat, Projector projector = null)
                             };
                             textureItem.TextureOnImport = () =>
                             {
-                                OpenFileDialog.Show(OnFileAccept, "Open image", Application.dataPath, FileFilter);
+#if !API
+                                string fileFilter = KK_Plugins.ImageHelper.FileFilter;
+#else
+                                string fileFilter = ""
```

Comment:
It's a pretty big regression for API builds.
```suggestion
                                string fileFilter = "Images (*.png;.jpg)|*.png;*.jpg|All files|*.*";
```